change/@ni-nimble-components-b8f24320-d835-42a0-b1d8-1b183206df94.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Capturing the plan discussed at standup:
We have a ton of bandwidth available to investigate these failures before the end of the year, but @msmithNI might have time next week. If we see more Firefox failures after this PR goes in then we should consolidate the bugs to a single issue and look into FF-specific fixes rather than test-specific fixes. Maybe we need to shard our FF tests or file memory leak bugs to Mozilla, for example. But for now it seems worth disabling these tests in the hope that the pipeline becomes more stable. Either way we left the bugs with the triage tag so we look at them in the new year.